examples/rh-sso/client-registrations/webhook_handler.lua

```diff
@@ -0,0 +1,105 @@
+local redis_pool = require 'client-registrations/redis_pool'
+local lom = require 'lxp.lom'
```

Comment:
the only issue I would have with slaxml is that it doesn't seem to be available from luarocks so I guess you'd have to clone the project instead